packages/@sanity/portable-text-editor/src/editor/Editable.tsx

```diff
@@ -1,5 +1,5 @@
-import {BaseRange, Transforms, Text, select, Editor} from 'slate'
-import React, {useCallback, useMemo, useEffect, forwardRef, useState, KeyboardEvent} from 'react'
+import {BaseRange, Transforms, Text, Editor} from 'slate'
```

Comment:
In the following files the change is to replace `useCallback` for `useCallbackWithTryCatch` and add details about the function.